examples/linear_model/plot_lasso_and_elasticnet.py

```diff
@@ -58,12 +57,13 @@
 print(enet)
 print("r^2 on test data : %f" % r2_score_enet)
 
-plt.plot(enet.coef_, color='lightgreen', linewidth=2,
-         label='Elastic net coefficients')
-plt.plot(lasso.coef_, color='gold', linewidth=2,
-         label='Lasso coefficients')
-plt.plot(coef, '--', color='navy', label='original coefficients')
+plt.stem(np.where(enet.coef_)[0], enet.coef_[enet.coef_ != 0], 'C1-',
```

Comment:
matplotlib 1 doesn't support 'C'

Reply:
@jnothman I have removed this. I was unable to pass the colors I wanted to plt.stem(), so I used a additional plt.setp() afterwards. I fear this is too complicated, what do you think ?